roles/kube_nfs_volumes/tasks/nfs.yml

```diff
@@ -3,6 +3,13 @@
   action: "{{ ansible_pkg_mgr }} name=nfs-utils state=present"
   when: not openshift.common.is_containerized | bool
 
+- name: Work around https://bugzilla.redhat.com/show_bug.cgi?id=1394395
+  file: path=/var/lib/nfs/{{ item }}
+  with_items:
+    - etab
+    - rmtab
+  when: openshift.common.is_atomic | bool
+
```

Comment:
I remember discussing this bug with either you or someone else and they said that we'd never support running NFS server on atomic host. Has that changed?